engine/src/main/scala/cromwell/webservice/routes/MetadataRouteSupport.scala

```diff
@@ -127,6 +129,14 @@ trait MetadataRouteSupport extends HttpInstrumentation {
 }
 
 object MetadataRouteSupport {
+
+  private def processWorkflowMetadataDeletedResponse(workflowId: WorkflowId, archiveStatus: MetadataArchiveStatus): JsObject = {
+    JsObject(Map(
+      WorkflowMetadataKeys.Id -> JsString(workflowId.toString),
+      WorkflowMetadataKeys.MetadataArchiveStatus -> JsString(archiveStatus.toString)
```

Comment:
I know I seagulled a "contact support" message in the mob last week, but I now think a brief prose cue could actually be helpful.

```suggestion
      WorkflowMetadataKeys.Id -> JsString(workflowId.toString),
      WorkflowMetadataKeys.MetadataArchiveStatus -> JsString(archiveStatus.toString),
      WorkflowMetadataKeys.MetadataArchiveStatusMsg -> JsString(
        "Cromwell archived this workflow's metadata according to the lifecycle policy. It is available in the archive bucket, or via a support request in the case of a managed instance."
      )
```

Reply:
I concur. I feel that a message would definitely be helpful. I will add that back. I guess, if in the future we think it's unnecessary, we can always remove it.